voting_tutorial/client_voting.py

```diff
@@ -0,0 +1,296 @@
+##########################################################################################
+
+#                                   SINGLE FILE VOTING
+
+##########################################################################################
+
+
+from pdb import set_trace as bp
+import argparse
+import asyncio
+import py_nillion_client as nillion
+import os
+import sys
+from dotenv import load_dotenv
+from config import (
+    CONFIG
+)
+
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
+from helpers.nillion_client_helper import create_nillion_client
+from helpers.nillion_keypath_helper import getUserKeyFromFile, getNodeKeyFromFile
+from digest_result import digest_plurality_vote_honest_result, digest_plurality_vote_dishonest_with_abort_result, digest_plurality_vote_robust_result
+
+load_dotenv()
+
+async def main():
+
+    cluster_id = os.getenv("NILLION_CLUSTER_ID")
+
+    while True:
+
+        # Below, you can choose which voting program to use. In case you choose a voting program 
+        # different from the robust version ('voting_dishonest_robust_6'), you can complete  
+        # either 'digest_plurality_vote_honest_result()' or 'digest_plurality_vote_dishonest_with_abort_result()' 
+        # functions above to digest the result.
+        #
+        # Existing voting programs:
+        #
+        # program_name = "voting_honest_1"  
+        # program_name = "voting_honest_2"  
+        # program_name = "voting_dishonest_abort_5"      
+        # program_name = "voting_dishonest_robust_6"
+
```

Comment:
Also not critical at all, is there a reason why this couldn't be called `voting_honest_1`, `voting_honest_2`, `voting_dishonest_abort_3` and `voting_dishonest_robust_4`.

Reply:
Those numbers are related with the program number on the voting tutorial .md page.